third_party/move/move-model/src/model.rs

```diff
@@ -1838,6 +1862,32 @@ impl GlobalEnv {
             .clone()
             .unwrap_or(Address::Numerical(AccountAddress::TWO))
     }
+
+    // Removes all functions not matching the predicate from
+    //   module_data fields function_data and function_idx_to_id
+    //   remaining function_data fields called_funs and calling_funs
+    pub fn filter_functions<F>(&mut self, mut predicate: F)
```

Comment:
Why do we need to do this and remove the inlined functions? There are scenarios (e.g. doc generation) where we don't want this. Wondering whats the rationale.

Reply:
The bytecode generator fails on the inline functions.